src/main/resources/assets/wurst/lang/zh_cn.json

```diff
@@ -65,52 +65,52 @@
-  "description.wurst.hack.mobspawnesp": "高亮那些生物可以生成的区域。\n§e黄色§r 可以在夜间生成\n§c红色§r - 可以在任意时候生成",
+  "description.wurst.hack.mobspawnesp": "高亮生物可以生成的区域。\n§e黄色§r 可以在夜间生成\n§c红色§r - 可以在任意时候生成",
   "description.wurst.hack.multiaura": "比 Killaura 更高级，能同一时间攻击多个实体。",
   "description.wurst.hack.nameprotect": "隐藏所有玩家的名字。",
   "description.wurst.hack.nametags": "改变名字标签大小使你可以经常可以看到他们，同时也能让你看到那些正潜行的玩家名字标签。",
   "description.wurst.hack.navigator": "一个可供你搜索的 GUI，可以随着时间的推移了解您的偏好。",
   "description.wurst.hack.nobackground": "移除GUI后的暗色背景。",
-  "description.wurst.hack.noclip": "允许你自由的穿过方块。\n一个（例如：沙子）可以掉落的方块掉在你的头顶才能激活。\n\n§c§l警告：§r 你将会受到伤害当你穿墙的时候！",
+  "description.wurst.hack.noclip": "允许你自由的穿过方块。\n一个（例如：沙子）可以掉落的方块掉在你的头顶才能激活。\n\n§c§l警告：§r 你会在你穿墙时受到伤害！",
   "description.wurst.hack.nocomcrash": "利用 Nocom 漏洞使服务器卡顿或崩溃。\n经测试可在 Vanilla、Spigot 和 Fabric 上工作，Paper 和一些拥有特定反作弊的服务器无法使用。",
   "description.wurst.hack.nofall": "使你从高处摔下来免受伤害。",
-  "description.wurst.hack.nofireoverlay": "防止火焰阻挡视野。\n\n§c§l警告：§r这可能会导致你，因不知道身上有火而烧伤致死。",
+  "description.wurst.hack.nofireoverlay": "防止着火时的第一人称视角下的火焰贴图阻挡视野。\n\n§c§l警告：§r这可能会导致你因不知道身上有火而烧伤致死。",
   "description.wurst.hack.nohurtcam": "关闭因受伤而产生视野摇晃效果。",
   "description.wurst.hack.nooverlay": "防止因在水中或岩浆产生视野阻挡。",
   "description.wurst.hack.nopumpkin": "防止因带南瓜头而导致视野阻挡。",
   "description.wurst.hack.noslowdown": "取消减速的效果。像蜂蜜、灵魂沙或者使用物品。",
-  "description.wurst.hack.noweather": "允许改变你当地客户端所在的时间，和月亮的阶段（仅在本地生效，与服务器无关）。",
+  "description.wurst.hack.noweather": "允许改变你本地客户端所在的时间，和月亮的阶段（仅在本地生效，与服务器无关）。",
   "description.wurst.hack.noweb": "使你不会因为蜘蛛网而慢下来。",
   "description.wurst.hack.nuker": "自动挖掘你附近的方块。",
   "description.wurst.hack.nukerlegit": "NukerLegit 能够绕过所有的反作弊检测。\n在正常的 NoCheat+（NCP）服务器上不需要！",
   "description.wurst.hack.openwateresp": "显示你是否在一个“开放水域”的地方，在区域内画一个大方块用于计算检测是否为开放水域。",
   "description.wurst.hack.overlay": "开采一个方块时渲染 “Nuker 功能” 动画。",
   "description.wurst.hack.panic": "瞬间关闭所有的作弊功能。\n请小心使用！",
-  "description.wurst.hack.parkour": "你会自动跳跃起来，当到方块的边缘。\n适用于跑酷或跳和跑。",
+  "description.wurst.hack.parkour": "当你到达方块边缘时自动跳起。\n适用于跑酷或部分需要跑跳的场景。",
   "description.wurst.hack.playeresp": "高亮透视附近的玩家。\n如果是你的好友的话会显示蓝色框框。",
   "description.wurst.hack.playerfinder": "在雷雨中寻找远处的玩家。",
   "description.wurst.hack.portalgui": "允许你在传送门内打开 GUI。",
   "description.wurst.hack.potionsaver": "当你站着不动时冻结所有的药水效果时间。",
-  "description.wurst.hack.prophuntesp": "在躲猫猫小游戏中知道哪些是人扮的方块。\n用于 Mineplex 服务器的 Prophunt（躲猫猫），其他服务器未必凑效。",
+  "description.wurst.hack.prophuntesp": "在躲猫猫小游戏中知道哪些是人扮的方块。\n用于 Mineplex 服务器的 Prophunt（躲猫猫），其他服务器未必奏效。",
   "description.wurst.hack.protect": "使用 AI 模式保护最近的实体，并保护它免受其他实体的伤害。\n使用指令 .protect 来保护指定的实体，而不是最近的一个。",
   "description.wurst.hack.radar": "生成一个雷达 UI 并显示附近的实体。\n§c红色§r - 玩家\n§6橘色§r - 怪物\n§a绿色§r - 动物\n§7灰色§r - 其他",
-  "description.wurst.hack.rainbowui": "§c让§a每§9一§c样§a都 §9充§c满§a五§9彩§c斑§a斓§9的§c样§a子§9，§c包§a括§9你§c当§a前§9所§c在 §aU§9I",
+  "description.wurst.hack.rainbowui": "§c让§aW§9u§cr§as§9t§cU§aI§c中§a的§c一§a切§c都§a五§9彩§c斑§a斓§9，§c包§a括§9你§c当§a前§9所§c在 §aU§9I",

```

Comment:
This seems redundant. As I understand this says something like:

>Makes everything colorful in the Wurst UI, including the UI.

Why is the UI is mentioned twice?

Reply:
Sorry, I forgot to delete the ”included UI“ when changing the source file